core/arch/aarch64/emit_utils.c

```diff
@@ -343,12 +352,25 @@ insert_fragment_prefix(dcontext_t *dcontext, fragment_t *f)
     /* Always use prefix on AArch64 as there is no load to PC. */
     byte *pc = (byte *)f->start_pc;
     ASSERT(f->prefix_size == 0);
-    /* ldr x0, [x(stolen), #(off)] */
-    *(uint *)pc = (0xf9400000 | (ENTRY_PC_REG - DR_REG_X0) |
-                   (dr_reg_stolen - DR_REG_X0) << 5 |
-                   ENTRY_PC_SPILL_SLOT >> 3 << 10);
-    pc += 4;
-    f->prefix_size = (byte)(((cache_pc)pc) - f->start_pc);
+    if (use_ibt_prefix(f->flags)) {
+        /* ldr x0, [x(stolen), #(off)] */
+        *(uint *)pc = (0xf9400000 | (ENTRY_PC_REG - DR_REG_X0) |
+                       (dr_reg_stolen - DR_REG_X0) << 5 |
+                       ENTRY_PC_SPILL_SLOT >> 3 << 10);
+        pc += 4;
+        f->prefix_size = (byte)((cache_pc)pc - f->start_pc);
+    } else {
+#ifdef CLIENT_INTERFACE
+        /* FIXME i#1569 */
```

Comment:
What does this FIXME refer to?  Is there something incomplete about the code below?  (i#1569 is very general so it doesn't help.  The comment should explain.)